tests/test_snapshots.py

```
@@ -328,3 +328,56 @@ def test_write_snapshots_handles_errors() -> None:
+            super().__init__()
+            self.cursors: dict[str, datetime] = {}
+
+        def set_cursor(self, source: str, cursor: datetime) -> None:
+            self.cursors[source] = cursor
+
+    storage = CursorStorage()
+    config = BotConfig(
+        runtime=RuntimeConfig(
+            mode=RunMode.DRY_RUN,
+            log_level="INFO",
+            data_dir="/tmp/test",
```

Ruff (0.16.0)

[error] Probable insecure usage of temporary file or directory: "/tmp/test"

(S108)